src/pages/ClaimsPage/Table.tsx

```diff
@@ -0,0 +1,72 @@
+import { ReactNode, useMemo, useState } from 'react';
```

Comment:
Can you move this to `src/components` and change `AllocationsTable` to use it as well, instead of duplicating code?